pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -162,6 +159,10 @@ func (alloc *Action) allocateResources(queues *util.PriorityQueue, jobsMap map[a
 			tasks.Len(), job.Namespace, job.Name)
 
 		alloc.allocateResourcesForTasks(tasks, job, jobs, queue, allNodes)
+
+		// Put back the queue to priority queue after job's resource allocating finished,
+		// To ensure that the priority of the queue is calculated based on the latest resource allocation situation.
```

Comment:
`To ensure that the priority of the queue is calculated based on the latest resource allocation`  refers to Line#122 or Line#124. I'd like to know what exactly the problem is. Would you please clarify more?